docs/versioning.md

```diff
@@ -20,8 +20,12 @@ Ruff uses a custom versioning scheme that uses the **minor** version number for
     - Stable rules are added to the default set
     - Stable rules are removed from the default set
     - A safe fix for a rule is promoted to stable
-- Formatter: 
+- Formatter:
      - The stable style changed
+- Language server:
+    - The behavior of an existing capability is changed
+        - The scope of an existing capability is significantly increased
```

Comment:
I think this is overly strict and at the same time not specific enough. Adding new lint rules changes the lint capability. 

I also think that it's fine to add new actions to the *code action* capability. 

I wonder if we should remove this for now and limit it to the removal of capabilities or settings.

Reply:
Yeah, I think that makes sense. It's unclear what kind of restrictions we want right now so I'll remove this part now.

Reply:
Thanks for the feedback!